foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -567,6 +591,8 @@ def retry_failed_processor_jobs() -> None:
         volume_index__in=active_volumes
     ).exclude(
         pipeline_applied="JANITOR"
```

Comment:
Is this exclude logic correct? Ex smashers, QN, etc

Reply:
This is about whether or not to retry the jobs. Janitor jobs get queued every half hour whether they worked or not, so I don't know that we need to worry about retrying them constantly. However if a smasher or QN job failed I do think they'd be worth retrying. I'm open to discussing it though if you don't agree.